pkg/api/unversioned/validation/validation.go

```diff
@@ -17,17 +17,22 @@ limitations under the License.
 package validation
 
 import (
+	"fmt"
+
 	"k8s.io/kubernetes/pkg/api/unversioned"
-	apivalidation "k8s.io/kubernetes/pkg/api/validation"
+	"k8s.io/kubernetes/pkg/util/validation"
 	"k8s.io/kubernetes/pkg/util/validation/field"
 )
 
+var labelValueErrorMsg string = fmt.Sprintf(`must have at most %d characters, matching regex %s: e.g. "MyValue" or ""`, validation.LabelValueMaxLength, validation.LabelValueFmt)
```

Comment:
Could you 

var ( 
    labelValueErrorMsg ...
) 

Also why strings vs. Errors, as these are clearly errors.